src/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarAcquireable.tsx

```diff
@@ -0,0 +1,69 @@
+import { Box, Button, Serif } from "@artsy/palette"
+import React from "react"
+import { createFragmentContainer, graphql } from "react-relay"
+
+import { ArtworkSidebarAcquireable_artwork } from "__generated__/ArtworkSidebarAcquireable_artwork.graphql"
+export interface ArtworkSidebarAcquireableProps {
+  artwork: ArtworkSidebarAcquireable_artwork
+}
+
+export const ArtworkSidebarAcquireable: React.SFC<
+  ArtworkSidebarAcquireableProps
+> = props => {
+  const { artwork } = props
+  return (
+    <>
+      <Box pb={2}>
+        {artwork.is_sold ? (
+          <Serif size="5t" weight="semibold">
+            Sold
+          </Serif>
+        ) : (
+          <>
+            <Serif size="5t" weight="semibold">
+              {artwork.price}
+            </Serif>
+            <Serif size="2" color="black60">
+              {artwork.shippingInfo}
+            </Serif>
+            <Serif size="2" color="black60">
+              Ships from {artwork.shippingOrigin}
```

Comment:
Also believe it's safer to display shippingInfo and shippingOrigin fields for everybody and let metaphysics decide if the works should show/hide this information. It actually already does. That is the pr you've added here: https://github.com/artsy/metaphysics/pull/1385

I think it's better if we can control data in MP and the client here is just displays the information. This way we can change the rules without needed to update UI.